packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { findDOMNode } from 'react-dom';
-import { BaseDecorator } from './BaseDecorator';
+import { BaseDecorator } from '../decorators/BaseDecorator';
```

Comment:
This line can be reverted now
```suggestion
import { BaseDecorator } from './BaseDecorator';
```